### PR TITLE
feat(ReceiveModal): add helper text to explain generated addresses

### PR DESCRIPTION
**Pull request by plondon**
screen shot 2016-02-01 at 5 56 46 pm

### CODE REVIEW

**Comment by ebepark** [comment]
Current: A new address *will be* generated...
Suggested: A new address *is* generated...

Current: Bitcoin Address for your next transaction (Capitalize: "Your Next Transaction")
Suggested: Receive Bitcoin Using This New Bitcoin Address

Current: Copy the above address **and **share with others!
Suggested: Copy the above address ** to** share with others!

**Comment by Sjors** [comment]
Code-wise it looks good to me.

File: assets/css/components/_helpers.scss [hunk 1]
```diff
@@ -15,9 +15,9 @@
     outline: none;
   }
   &.active {
+    border: 1px solid transparent;
```
Comment (plondon) [inline assets/css/components/_helpers.scss]: Always transparent border over border: 0px to avoid page jumpiness.

**Comment by jtormey** [comment]
Looks good

**Pull request merged**